src/EventGrid/EventGrid/Domain/GetAzureEventGridDomain.cs

```diff
@@ -89,8 +138,31 @@ public override void ExecuteCmdlet()
             {
                 resourceGroupName = this.ResourceGroupName;
             }
+            else if (!string.IsNullOrEmpty(this.NextLink))
```

Comment:
@ahamad-MS is this if-statement necessary? It seems that you should be able to replace the usage of `nextLink` with `NextLink` in the code below.

(this comment applies to all instances of this in the PR)

Reply:
OK. Fixed.

Reply:
Fixed.